samples/grpc/local-drone-control-scala/src/main/resources/prod.conf

```diff
@@ -0,0 +1,39 @@
+# Production configuration for running the local-drone-control service in Kubernetes,
+# as a multi-node cluster and with a separate PostgreSQL database.
+
+include "cluster"
+include "grpc"
+include "postgres"
+
+local-drone-control {
+  # consider setting this to a specific interface for your environment
+  grpc.interface = "0.0.0.0"
+  grpc.interface = ${?GRPC_INTERFACE}
+
+  nr-of-event-producers = 4
+  # unique identifier for the instance of local control, must be known up front by the cloud service
+  location-id = "sweden/stockholm/kungsholmen"
+  location-id = ${?LOCATION_ID}
+
+  ask-timeout = 3s
+}
+
+akka.remote.artery {
+  canonical.port = 2552
+  canonical.port = ${?REMOTE_PORT}
+}
+
```

Comment:
How does this config file compare to other clustered samples? e.g. restaurant-drone-deliveries-service-scala
I'm missing for example downing-provider-class

Reply:
Ok. I'll make sure it has everything. And it could make sense to consolidate into the `cluster.conf` file for when it's clustered, (and not included from single-node `application.conf`).